### PR TITLE
🌐 Add Chinese translation for Tutorial - CORS (Cross-Origin Resource Sharing)

### PR DESCRIPTION
**Pull request by blt232018**
CORS (Cross-Origin Resource Sharing) 章节的中文翻译。
@waynerv @maoyibo and  any other friends who feels free to help review this chapter. Thx you so much!

### CODE REVIEW

File: docs/zh/docs/tutorial/cors.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+* `https://localhost`
+* `http://localhost:8080`
+
+即使它们都在 `localhost` 中，但是它们使用不同的协议或者端口，所以它们都是不同的「源」。
+
+## 步骤
+
+假设你的浏览器中有一个前端运行在 `http://localhost:8080`，并且它的 JavaScript 正在尝试与运行在 `http://localhost` 的后端通信（因为我们没有指定端口，浏览器会采用默认的端口 `80`）。
+
+然后，浏览器会向后端发送一个 HTTP `OPTIONS` 请求，如果后端发送适当的 headers 来授权来自不同源（`http://localhost:8080`）的通信，浏览器将允许前端的 JavaScript 向后端发送请求。
```
Comment (waynerv) [inline docs/zh/docs/tutorial/cors.md:21]:
```suggestion
然后，浏览器会向后端发送一个 HTTP `OPTIONS` 请求，如果后端发送适当的 headers 来授权来自这个不同源（`http://localhost:8080`）的通信，浏览器将允许前端的 JavaScript 向后端发送请求。
```

File: docs/zh/docs/tutorial/cors.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+你可以在 **FastAPI** 应用中使用 `CORSMiddleware` 来配置它。
+
+* 导入 `CORSMiddleware`。
+* 创建一个允许的源列表（字符串类型）。
+* 将其作为「中间件」添加到你的 **FastAPI** 应用中。
+
+你也可以指定后端是否允许：
+
+* 凭证（授权 headers，Cookies 等）。
+* 特定的 HTTP 方法（`POST`，`PUT`）或者使用通配符 `"*"`。
```
Comment (waynerv) [inline docs/zh/docs/tutorial/cors.md:46]:
```suggestion
* 特定的 HTTP 方法（`POST`，`PUT`）或者使用通配符 `"*"` 允许所有方法。
```

File: docs/zh/docs/tutorial/cors.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+* 特定的 HTTP headers 或者使用通配符 `"*"`。
```
Comment (waynerv) [inline docs/zh/docs/tutorial/cors.md:47]:
```suggestion
* 特定的 HTTP headers 或者使用通配符 `"*"` 允许所有 headers。
```

File: docs/zh/docs/tutorial/cors.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+但这仅允许某些类型的通信，不包括所有涉及凭据的内容：像 Cookies 以及那些使用 Bearer 令牌的授权 headers 等。
+
+因此，为了一切都能正常工作，最好显式地指定允许的源。
+
+## 使用 `CORSMiddleware`
+
+你可以在 **FastAPI** 应用中使用 `CORSMiddleware` 来配置它。
+
+* 导入 `CORSMiddleware`。
+* 创建一个允许的源列表（字符串类型）。
```
Comment (waynerv) [inline docs/zh/docs/tutorial/cors.md:40]:
```suggestion
* 创建一个允许的源列表（由字符串组成）。
```

File: docs/zh/docs/tutorial/cors.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+* 凭证（授权 headers，Cookies 等）。
+* 特定的 HTTP 方法（`POST`，`PUT`）或者使用通配符 `"*"`。
+* 特定的 HTTP headers 或者使用通配符 `"*"`。
+
+```Python hl_lines="2  6-11  13-19"
+{!../../../docs_src/cors/tutorial001.py!}
+```
+
+默认情况下，`CORSMiddleware` 的默认参数实现受限，所以你需要显式地启用特定的源、方法或者 headers，以便浏览器能够在跨域上下文中使用它们。
```
Comment (waynerv) [inline docs/zh/docs/tutorial/cors.md:53]:
```suggestion
默认情况下，这个 `CORSMiddleware` 实现所使用的默认参数较为保守，所以你需要显式地启用特定的源、方法或者 headers，以便浏览器能够在跨域上下文中使用它们。
```

File: docs/zh/docs/tutorial/cors.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+* `expose_headers` - 指示可以被浏览器访问的响应头。默认为 `[]`。
+* `max_age` - 设定浏览器缓存 CORS 响应的最长时间，单位是秒。默认为 `600`。
+
+中间件响应两种特定类型的 HTTP 请求……
+
+### CORS 预检请求
+
+这是些带有 `Origin` 和 `Access-Control-Request-Method` 请求头的 `OPTIONS` 请求。
+
+在这种情况下，中间件将拦截传入的请求并进行响应，返回一个使用适当的 CORS headers 的 `200` 或 `400` 响应，出于提供信息的目的。
```
Comment (waynerv) [inline docs/zh/docs/tutorial/cors.md:71]:
```suggestion
在这种情况下，中间件将拦截传入的请求并进行响应，出于提供信息的目的返回一个使用了适当的 CORS headers 的 `200` 或 `400` 响应。
```

File: docs/zh/docs/tutorial/cors.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+### 简单请求
+
+任何带有 `Origin` 请求头的请求。在这种情况下，中间件将像平常一样传递请求，但是在响应中包含适当的 CORS headers。
+
+## 更多信息
+
+更多关于 <abbr title="Cross-Origin Resource Sharing">CORS</abbr> 的信息，请查看 <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS" class="external-link" target="_blank">Mozilla CORS documentation</a>。
```
Comment (waynerv) [inline docs/zh/docs/tutorial/cors.md:79]:
```suggestion
更多关于 <abbr title="Cross-Origin Resource Sharing">CORS</abbr> 的信息，请查看 <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS" class="external-link" target="_blank">Mozilla CORS 文档</a>。
```

**Review by waynerv — COMMENTED** [review]
Great translation!

**Comment by blt232018** [comment]
thx for review!

> Great translation!

**Comment by tiangolo** [comment]
Thanks for your contribution! The GitHub Actions CI errors were related to a change in the way pip works underneath, not to your PR. You should be able to merge or rebase from the `master` branch and it should be fixed. :heavy_check_mark:

**Pull request closed without merging**